src/components/Identity/index.js

```diff
@@ -88,11 +88,19 @@ const createIdentity = ({ config, logger, cookie }) => {
           const timestamp = parseInt(cookie.get(ID_SYNC_TIMESTAMP) || 0, 36);
 
           if (config.idSyncsEnabled && nowInHours > timestamp) {
-            event.mergeQuery({
+            const identityQuery = {
               identity: {
                 exchange: true
               }
-            });
+            };
+
+            const containerId = parseInt(config.idSyncContainerId, 10);
+
+            if (!Number.isNaN(containerId)) {
```

Comment:
IE does not support `Number.isNaN` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

Reply:
Thanks.

Reply:
I'm getting
error  Unexpected use of 'isNaN'  no-restricted-globals
Should I edit eslint? It only accepts Number.isNaN, which doesn't exist in IE

Reply:
That sounds appropriate to me.